docker-compose.yml

```diff
@@ -7,7 +7,7 @@ services:
       target: dev
     volumes:
       - .:/app
-      - .env.docker:/app/.env
+      - .env:/app/.env
```

Comment:
AggregateError. Server is crashing isn't it?

Reply:
It's working well on my end.

---

<img width="482" alt="image" src="https://github.com/user-attachments/assets/4ad24864-4630-46a1-bc94-a86ec9542a49">
<img width="1348" alt="image" src="https://github.com/user-attachments/assets/07b10414-93bf-4d3b-8edf-a10ac24b2f5e">